detox/e2e/test/autocomplete/search.e2e.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {toChannelScreen} from '@support/ui/screen';
+
+import {Setup} from '@support/server_api';
+
+describe('Autocomplete', () => {
+    beforeAll(async () => {
+        const {user} = await Setup.apiInit();
+        await toChannelScreen(user);
+    });
+
+    beforeEach(async () => {
+        await device.reloadReactNative();
+
+        // # Enter search screen
+        await element(by.id('channel.search.button')).tap();
+    });
+
+    it('MM-T3393_1 should render at_mention component', async () => {
+        await expect(element(by.id('autocomplete.at_mention.list'))).not.toExist();
+
+        // # Tap "from:" modifier
+        await element(by.id('search_from')).tap();
+        await expect(element(by.id('autocomplete.at_mention.list'))).toExist();
```

Comment:
Just minor. Please add expect comments in this file similar to the other ones.

Reply:
Just added the comments! https://github.com/mattermost/mattermost-mobile/pull/4855/commits/70cd3bca5a7a66cb808f49889e5aca48c17e0167